### DISCUSSION TITLE
Outpost bots should make reactions to dragging a corpse

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15064] [Ideas and suggestions]
Currently if you drag a corpse of outpost dweller, witnesses do not make a reaction

**Comment by itchyOwl** [comment]
Yeah I agree. I'll create a ticket, but no promises about if/when this will be implemented.

**Comment by jigardi** [comment]
Have my upvote.
How widespread is this problem? Is the problem the same for corpses of crew in the submarine?
This could potentially be very useful for future missions requiring a mixture of stealth and violence.

**Comment by Regalis11** [comment]
Addressed in our private repo now, fix coming up in the next unstable build and the upcoming release.
